README.md

```diff
@@ -23,7 +23,7 @@ Maven:
 
 Gradle:
 ```
-implementation "io.pinecone:pinecone-client:1.2.2"
+implementation "io.pinecone:pinecone-client:2.0.0"
```

Comment:
Should this be `2.1.0`?

Reply:
So this was a typo from the last time and going forward, I'm planning to follow python SDKs format where we only update the versions in the README for major releases. Also I'll be adding a CI job to automate publishing.